### ISSUE TITLE
game no response. unable get into the game

### ISSUE THREAD

**Issue by zths2004** [derekhe/msfs2020-google-map#96]
when you start the program and enter the game, the game is stuck on the loading page. It works a few days ago but since yesterday it doesn't. If i close the program and run the game, i can easily enter the game...
Plz help me thx

**Comment by derekhe** [comment]
You can have a try to open msfs2020 first and then the mod.

**Comment by CastorSmith89** [comment]
I observe the same. I did not expect that this is related to this mod. I just kill the MSFS process and restart. The second run is then starting fine.

**Comment by derekhe** [comment]
Alright, thanks for feedback

**Comment by run-up15** [comment]
I have the same problem. Loading MSFS always gets stuck at the same point.
![Screenshot (2)](https://user-images.githubusercontent.com/94805297/142778218-5eb1bdaf-431d-4e6c-ac47-7130ea3205f2.png)

**Comment by run-up15** [comment]
![Screenshot (3)](https://user-images.githubusercontent.com/94805297/142778423-4f7b6367-513d-4bbe-90c7-8211fb9bb64c.png)

**Comment by CastorSmith89** [comment]
@run-up15 i had the same issue again after the goty update. I had an pending update in the ms store for c++. I also deleted the mod and used a fresh download of this mod. After start of msfs with the mod I cleared the msfs cache as well. Checking the ms store and replace instead of copy latest version to existing mod resolved my issue.

**Comment by frapell** [comment]
I can confirm this. Trying the mod on GOTY edition, and the first loading screen halts at about 50% and have to kill the game. Need to start the game first, and then click run in the mod.

**Comment by run-up15** [comment]
The problem is still present in the current version 2021-11-24. I wait until the "check for updates" is completed when starting MSFS and then press "RUN" so that the game works with the mod. Thanks for your tips

**Comment by derekhe** [comment]
Please try latest version: https://github.com/derekhe/msfs2020-google-map/releases/tag/2.3.2

**Comment by CastorSmith89** [comment]
Today I run again in this situation. What I observe is that I don't get ask to trust the certs. Is that maybe the problem that the message to trust the hand signed certs does not come up?

**Comment by derekhe** [comment]
Maybe

**Comment by MichaelPortelas** [comment]
I have same probleme, for fix delete all sub-folder in cache folder

**Comment by CastorSmith89** [comment]
Did not help. I deleted the complete mod. Rebooted and started without success. I started the mod on the main menu of msfs. I will check if it has an negative impact. 

At the end it is just a visual mod. I hope the creator of this great mod is not too upset with our issue reports 😂

**Comment by JuanJNovoa** [comment]
Same issue here. Manually deleted the certificate, started mod and got prompted to accept cert which always works when I see the cert windows, but tried the same process a second time and no cert accept window came up and it just hung. Something is not happening which may not have anything to do with his code, I just don't know what.

**Comment by JJpilot** [comment]
Same issue here. Manually deleted the certificate, started mod and got prompted to accept cert which always works when I see the cert windows, but tried the same process a second time and no cert accept window came up and it just hung. Something is not happening which may not have anything to do with his code, I just don't know what. I love the mod!!!!

**Comment by Khanon** [comment]
I have the same issue:

- Run the mod
- Start MSFS
- The game stuck on 50% loading process (CPU stop working hard)
- Close MSFS
- Stop and close the mod
- Run the mod again
- Start MSFS on normal mode
- The game loads and works fine, but sometimes I have to retry the process for third or fourth time.

No cert dialog at any point. Certs doesn't seem to be the problem.
I'm not sure what can be the problem but I'm not confident of restarting MSFS before playing.
Maybe a connection issue? I have very bad internet.

**Comment by JJpilot** [comment]
@Khanon, have you tried starting Run.bat, starting the sim before clicking on the Run button. Once sim starts, click Run?

**Comment by Khanon** [comment]
@JJpilot yes but in that way the mod doesn't download any Google texture.

**Comment by JJpilot** [comment]
@Khanon Sounds like your issue is different than mine.
